### ISSUE TITLE
Publishing to npm

### ISSUE THREAD

**Issue by jimfleming** [james2doyle/grunt-jsfmt#1]
Hey, I was wondering if this was already published and taken down or just not published yet. If jsfmt ends up using Grunt (there's some push to use Make instead) I was hoping to use this to format jsfmt.

**Comment by james2doyle** [comment]
No I havent published it yet. I was waiting for jsfmt to stabilize a little first. We were talking about exposing options when using require. Has that made any progress yet?

**Comment by jimfleming** [comment]
Yup, working on it now. Cool, good to know I didn't just miss it in npm.

**Comment by james2doyle** [comment]
Yeah I wanted to make sure that I didn't publish something that would be broke a day later. I'll watch the repo to keep up to date.

**Comment by aarongreenlee** [comment]
Do you think it has reached a stable point? 

Thanks for putting this task together.

**Comment by james2doyle** [comment]
I just need to update it to the latest version, then implement the config options. When I first made this there was no such feature.

**Comment by wavebeem** [comment]
Hey @james2doyle!

I'm looking into using this for a project I'm working on; it seems really promising. Is there anything you could use a hand with for getting this on **npm**?

**Comment by james2doyle** [comment]
Yeah actually. If you could update the project to the latest version of the
library that would be great. There are some features that are missing that
have been added since this started

On Tuesday, August 5, 2014, Brian Mock notifications@github.com wrote:

> Hey @james2doyle https://github.com/james2doyle!
> 
> I'm looking into using this for a project I'm working on; it seems really
> promising. Is there anything you could use a hand with for getting this on
> _npm_?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/james2doyle/grunt-jsfmt/issues/1#issuecomment-51274880
> .

## 

_James Doyle_

_t:  @james2doyle https://twitter.com/james2doylew: ohdoylerules.com
http://ohdoylerules.com_

**Comment by pagameba** [comment]
Hi, any progress on getting this to npm?

**Comment by james2doyle** [comment]
Once I get this up-to-date it should be good to go
